index.html

```diff
@@ -8,6 +8,7 @@
       rel="stylesheet"
       href="https://fonts.googleapis.com/css?family=Roboto"
     />
+    <title>Live Server Example</title>
```

Comment:
I missed this last time, but this is the second title that's on this index page.